kore/src/Kore/Variables/Free.hs

```diff
@@ -129,14 +115,20 @@ free variables as a synthetic attribute.
  -}
 synthetic
     :: Ord variable
-    => PatternF variable (Set.Set variable)
-    -> Set.Set variable
+    => PatternF variable (Set.Set (SubstVar variable))
+    -> Set.Set (SubstVar variable)
 synthetic (ExistsF Exists { existsVariable, existsChild }) =
-    Set.delete existsVariable existsChild
+    Set.delete (RegVar existsVariable) existsChild
 synthetic (ForallF Forall { forallVariable, forallChild }) =
-    Set.delete forallVariable forallChild
+    Set.delete (RegVar forallVariable) forallChild
 synthetic (VariableF variable) =
-    Set.singleton variable
+    Set.singleton (RegVar variable)
+synthetic (MuF Mu { muVariable = SetVariable muVar, muChild }) =
+    Set.delete (RegVar muVar) muChild
```

Comment:
SetVar? (same below). Also, test.

Reply:
Errors like this are now caught by the refactoring

Reply:
Apparently synthetic was not used anywhere, so I removed it.